share/rocm/cmake/ROCMSetupVersion.cmake

```diff
@@ -154,3 +154,12 @@ function(rocm_setup_version)
     endif()
 
 endfunction()
+
+function(rocm_set_soversion)
+    set(oneValueArgs LIBRARY_TARGET SOVERSION)
+
+    cmake_parse_arguments(PARSE "${options}" "${oneValueArgs}" "${multiValueArgs}" ${ARGN})
```

Comment:
There is actually no need to use `cmake_parse_arguments`. I think doing `function(rocm_set_soversion LIBRARY_TARGET SOVERSION)` would work and should be clear enough when called(ie `rocm_set_soversion(rocfft 0.1)`

Reply:
Without the parse, both variables were not usable in the function.

Reply:
I had to add the parse after trying without

Reply:
I see what you are asking, fixing it.